src/index.js

```diff
@@ -102,6 +102,8 @@ module.exports = plugin.withOptions(
         ['th'],
         ['td'],
         ['img'],
+        ['picture'],
+        ['img', 'picture > img'],
```

Comment:
This should be `images`?

```suggestion
        ['images', 'img', 'picture > img'],
```